bcgov/moh-prime
# PRIME-2099 Cannot submit Device Provider

### *** awaiting workflow confirmation from client

[Jira Ticket](https://bcgovmoh.atlassian.net/jira/software/c/projects/PRIME/boards/1?modal=detail&selectedIssue=PRIME-2099)

### How to test:

Try to submit different user enrolments using the following scenarios:

1. Select Device Provider in Care Settings and provide both Device Provider Identifier  and College Certificate in Regulatory page
2. Select Device Provider in Care Settings and provide Device Provider Identifier only in Regulatory Page (you will be forced to provide OBO sites)
3. Select Device Provider in Care Settings and provide College Certificate only in Regulatory Page (you will be forced to provide OBO sites)
4. Select Device Provider in Care Settings and provide nothing in Regulatory Page (you will be forced to provide OBO sites)
5. Select any Care Setting type other than Device Provider

You should be able to complete the submission in all of the above cases.
